service/main.tf

```diff
@@ -1,5 +1,9 @@
 #tfsec:ignore:aws-vpc-no-public-ingress-sgr tfsec:ignore:aws-vpc-no-public-egress-sgr
 module "ecs_alb_service_task" {
+  #checkov:skip=CKV_AWS_249:Roles are passed in
+  #checkov:skip=CKV_AWS_97:Transit encryption is optional and can be enabled by passing in configuration to the cloudposse module
+  #checkov:skip=CKV_AWS_111:Details of LB and TGs are unknown until the app resources are created
```

Comment:
This one is to do with:
`Ensure IAM policies does not allow write access without constraints`

Could you please address it?

Reply:
This policy is provided by the cloudposse module and so it wouldn't be super easy to just swap it out.

The policy is as follows:
```
  statement {
    effect    = "Allow"
    resources = ["*"]

    actions = [
      "elasticloadbalancing:Describe*",
      "elasticloadbalancing:DeregisterInstancesFromLoadBalancer",
      "elasticloadbalancing:RegisterInstancesWithLoadBalancer",
      "ec2:Describe*",
      "ec2:AuthorizeSecurityGroupIngress",
      "elasticloadbalancing:RegisterTargets",
      "elasticloadbalancing:DeregisterTargets"
    ]
  }
```
I think that Checkov is warning on this simply due to the wildcard in the resources list. 

The checkov [guide](https://docs.bridgecrew.io/docs/ensure-iam-policies-do-not-allow-write-access-without-constraint) links out to https://cloudsplaining.readthedocs.io/en/latest/glossary/resource-exposure/ Which contains a list of actions that are considered to be "Resource Exposure" actions. The above policy doesn't contain any of the actions which would cause `CKV_AWS_111` to be a concern. Do you think this still needs addressing?

Reply:
No, I think this is fair enough.